Assets/Scripts/DraggableItem.cs

```diff
@@ -0,0 +1,102 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class DraggableItem : MonoBehaviour
```

Comment:
- Je gebruikt geen namespace (dit doen we om eventuele conflicts te voorkomen)
 - Geen class summary (leg kort uit wat deze code doet, zodat iedereen weet wat hij doet)

Reply:
`namespace DN.UI
{

}`